src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/SymbolKey/SymbolKey.PropertySymbolKey.cs

```diff
@@ -20,44 +20,88 @@ public static void Create(IPropertySymbol symbol, SymbolKeyWriter visitor)
             public static SymbolKeyResolution Resolve(SymbolKeyReader reader, out string? failureReason)
```

Comment:
the majority of this change is bringing property rsolution in line with method resolution.